fetch-inputs.py

```diff
@@ -33,8 +33,9 @@
     sys.exit(1)
 
 SERVER_ADDRESS = os.getenv("RUST_DIST_SERVER", "https://static.rust-lang.org")
+CARGO_SERVER_ADDRESS = os.getenv("CARGO_DIST_SERVER",
+        "https://s3-us-west-1.amazonaws.com/rust-lang-cargo-dev/cargo-master")
```

Comment:
What does the naming of this bucked signify? It looks like a "dev" bucket for "master", but doesn't this script fetch from a production bucket for other branches than master?